send.c

```diff
@@ -594,6 +596,12 @@ static struct safe_buffer_list *add_ra_options_rdnss(struct safe_buffer_list *sb
 
 		memset(&rdnssinfo, 0, sizeof(rdnssinfo));
 
+		size_t const bytes = sizeof(rdnssinfo) + sizeof(struct in6_addr) * rdnss->AdvRDNSSNumber;
+		if (bytes > (256 * 8)) {
```

Comment:
Why did you pick 2048? 
- 1280 bytes is the min IPv6 MTU; what would happen in that case?
- can radvd safely send multiple RA packets with different RDNSS entries? (other framework code for example splits large options into their own packets)

Reply:
in my opinion
1、The IPv6 MTU value may change the processing of the kernel protocol stack.
2、RDNSS 2048 is consistent with DNSSL packet maximum in radvd

Do you have any good suggestions for me?